config/manager/kustomization.yaml

```diff
@@ -5,6 +5,6 @@ kind: Kustomization
 images:
 - name: controller
   newName: skyscanner/kms-issuer
-  newTag: latest
+  newTag: dev
```

Comment:
Can you add a not explaining why we're moving from `latest` to `dev`?

Reply:
I've added a note in the PR

> changing default container tag from `latest` (Docker default) to `dev` to avoid potential future problems